### ISSUE TITLE
`crates/jit/src/unwind/winx32.rs` doesn’t compile

### ISSUE THREAD

**Issue by KaDiWa4** [bytecodealliance/wasmtime#3634]
Currently, the file `crates/jit/src/unwind/winx32.rs` serves no purpose. In #3180 the other two unwind implementations (`winx64.rs` and `systemv.rs`) were rewritten, but the stub in `winx32.rs` was left unchanged. This means that the crate doesn't compile on 32-bit Windows. The file could be replaced with a new stub or removed entirely, I don't know.

**Comment by bjorn3** [comment]
Cranelift currently doesn't have an 32bit x86 backend. Only x86_64 and AArch64 are currently supported. See https://github.com/bytecodealliance/wasmtime/issues/1980.

**Comment by KaDiWa4** [comment]
Then I guess it would make sense to remove the file so it shows the [unsupported-platform error][source] when trying to compile.

[source]: https://github.com/bytecodealliance/wasmtime/blob/dd85d66ec353959479f7d8518a79d0c7d0b9e3ad/crates/jit/src/unwind.rs#L12

**Comment by alexcrichton** [comment]
Yes I think this code is historical and may have worked at one point but no longer does. I think it would be fine to move it to an unsupported platform as you've found.

**Comment by kadiwa4** [comment]
Closed by #5226.
